src/offline/cable_mpimaster.F90

```diff
@@ -7619,7 +7829,7 @@ SUBROUTINE master_casa_dump_types(comm, casamet, casaflux, phen, climate )
             &                             types(bidx), ierr)
        blocks(bidx) = 1
 
-       ! #294 - Avoid malformed var write for now 
+       ! #294 - Avoid malformed var write for now  ! should this block be uncommented - rk4417 - phase2
```

Comment:
> Why should it be uncommented?

Most likely because I noticed that it was uncommented in MMY code, otherwise I would not have left this remark. I left it commented out and removed my question.